.github/workflows/bench.yml

```diff
@@ -0,0 +1,68 @@
+name: benches
+
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Should we run this on every pull request? It should be okay to just run on pushing commits to dev branch imho.

Reply:
Your right lets remove PR